### DISCUSSION TITLE
Display using theme clear or dark

### DISCUSSION THREAD

**Discussion by pzorba75** [sumatrapdfreader/sumatrapdf#4003] [General]
I've been using sumatrapdf for years with a white theme. Suddenly, all displays have been moved in dark, regardless of my setteings in Prefrerences themes and selecting "Clair' (in french).
I'm not able to restore the white mode, I've reinstalled completely sumatra and the dark mode remains the only page layout.
What can I do to continue to work fine with whit theme?
Thanks for your help.
Pierre

**Comment by GitHubRulesOK** [comment]
There should not be such a problem so in settings it should work
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/30c73dd7-6841-42fd-be9a-eb378e0b9ce8)

If it is Dark I can edit in preferences
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/1e2512fa-7164-4e26-a52a-9118c0d6ed0c)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/ce951eee-c07a-44c4-a21a-1758a2fe2a06)


so **exactly** which version are you using ?

**Reply by pzorba75** [reply]
I was using the latest version 3.5.2 64-bit. Problem fixed by pressing **i** onto the page displayed. Miracle solution given by D. Bitouze a Latex  user supporting SumatraPDF.. 
Thanks for your help.
